docs/1-Using-Fleet/3-REST-API.md

```diff
@@ -17,6 +17,18 @@
   - [Create a user account with an invitation](#create-a-user-account-with-an-invitation)
   - [Create a user account without an invitation](#create-a-user-account-without-an-invitation)
   - [Get user information](#get-user-information)
+- [Queries](#queries)
+  - [Get query](#get-query)
+  - [List queries](#list-queries)
+  - [Create query](#create-query)
+  - [Modify query](#modify-query)
+  - [Delete query](#delete-query)
+  - [Delete query by ID](#delete-query-by-id)
+  - [Get queries specs](#get-quieries-specs)
+  - [Get query spec](#get-query-spec)
+  - [Apply queries specs](#apply-quieries-specs)
+  - [Create distributed query campaign](#create-distributed-query-campaign)
```

Comment:
I wonder if we should just call these "live queries"? Mostly that is the terminology folks use these days.

Reply:
Let's do live queries 👍